packages/benchmarks/.env.example

```diff
@@ -18,3 +18,7 @@ OPENAI_API_KEY="<api_key>"
 MONGODB_AUTH_COOKIE="<Auth cookie from CorpSecure-protected site>"
 GCP_OPENAI_ENDPOINT="https://{LOCATION}-aiplatform.googleapis.com/v1beta1/projects/{PROJECT_ID}/locations/{LOCATION}/endpoints/openapi"
 GCP_API_KEY="SOME_API_KEY"
+AWS_REGION="us-east-1"
+AWS_ACCESS_KEY_ID="..."
+AWS_SECRET_ACCESS_KEY="..."
+AWS_SESSION_TOKEN="note: must refresh regularly"
```

Comment:
Is the token something we could fetch programmatically through e.g. STS?

https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/sts/command/GetSessionTokenCommand/

Reply:
theres def some other ways we could get this token, which we'd wanna use for longer runnign tasks.

i think the current default token expiration time is like 1 hour